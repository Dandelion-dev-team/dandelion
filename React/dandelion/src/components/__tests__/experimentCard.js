import React from "react"
import renderer from "react-test-renderer"
import ExperimentCard from "../experimentCard"

jest.mock("../../auth");

describe("Experiment Card", () => {
  it("renders correctly", () => {
    const testProp = { 'id': '1', 'project_image_link': 'testlink', 'title': 'Test Title', "description": "Test description" };
    const tree = renderer
      .create(<ExperimentCard dataProp={testProp}/>)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})