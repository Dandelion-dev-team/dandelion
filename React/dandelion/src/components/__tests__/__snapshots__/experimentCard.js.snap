// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Experiment Card renders correctly 1`] = `
<div
  className="experiment-card"
>
  <div
    className="card-img"
  >
    <img
      src="testlink"
    />
  </div>
  <div
    className="title"
  >
    <h2>
      Test Title
    </h2>
  </div>
  <div
    className="center"
  >
    <div
      class="vl"
    />
  </div>
  <div
    className="description"
  >
    <h2>
      Test description
    </h2>
  </div>
</div>
`;
